packages/universal-components/src/Button/Button.stories.js

```diff
@@ -37,20 +37,21 @@ storiesOf('Button', module)
       ],
       'primary',
     );
+    const size = select('Size', ['small', 'normal', 'large'], 'normal');
     const disabled = boolean('Disabled', false);
     let sublabel;
     if (Platform.OS !== 'web') {
-      sublabel = text('Sublabel', 'Sublabel');
```

Comment:
Reason for this change?

Reply:
For faster development, not to remove it after each change. Also changed icons. Should I change it back?

Reply:
I like when initial Playground state shows all component possibilities. So I'm for setting it back. But we can keep new variable icons 😉. Thanks

Reply:
Its back :)